Microsoft.Toolkit.Uwp.UI/Converters/StringFormatConverter.cs

```diff
@@ -18,29 +19,34 @@ public class StringFormatConverter : IValueConverter
         /// <param name="value">Object to transform to string.</param>
         /// <param name="targetType">The type of the target property, as a type reference</param>
         /// <param name="parameter">An optional parameter to be used in the string.Format method.</param>
-        /// <param name="language">The language of the conversion (not used).</param>
+        /// <param name="language">The language of the conversion. If language is null or empty then <see cref="CultureInfo"/> will be used.</param>
         /// <returns>Formatted string.</returns>
         public object Convert(object value, Type targetType, object parameter, string language)
         {
-            if (value == null)
+            if (value is null)
             {
                 return null;
             }
 
-            if (parameter == null)
+            // Retrieve the format string and use it to format the value.
+            string formatString = parameter as string;
+            CultureInfo culture = string.IsNullOrWhiteSpace(language) ? CultureInfo.InvariantCulture : new CultureInfo(language);
```

Comment:
We should move this to within the try/catch L37 and only construct this value if we have a format string, otherwise we're doing this work when we don't need to.